CONTRIBUTING.md

```diff
@@ -199,5 +199,4 @@ are ready to review, as GitHub does not generate a notification when
 you git push.
 
 ## Final Words
```

Comment:
Was this accidental line removal?

Reply:
Yes, sorry about that

Reply:
@thomsmoreau , just about this line removal, was this intentional?

Reply:
Not at all, it was accidental

Reply:
Ok, so could you check the Files Changed section and revert the empty lines added/removed. I will put a comment in the other one as well.